### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by krasimir**
Here we have a description of the PR.

### CODE REVIEW

File: Content.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Content({ children }) {
+  return (
+    <section>
+      { children }
```
Comment (krasimir) [inline Content.js:10]: Maybe wrap that into a `<div>`. Like for example:

```suggestion
<div>
      { children }
</div>
```

File: Header.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 import React from 'react';
 
 export default function Header({ children }) {
-  return <h1>{ children }</h1>;
+  return (
+    <header>
+      <h1>{ children }</h1>
```
Comment (krasimir) [inline Header.js]: What about using `<h2>`?

Comment (krasimir) [reply Header.js]: Or maybe keep `<h1>`.

File: README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+A repo for testing while developing GitHorn
```
Comment (krasimir) [inline README.md:1]: Can you add more text here. !

File: Content.js (hunk 1 above)
Comment (stefantsonev2019) [reply Content.js:10]: I agree.

File: Header.js [hunk 4]
```diff
@@ -1,5 +1,10 @@
 import React from 'react';
 
 export default function Header({ children }) {
-  return <h1>{ children }</h1>;
+  return (
+    <header>
```
Comment (stefantsonev2019) [inline Header.js]: Shall we create a dedicated component here.

File: Content.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (stefantsonev2019) [inline Content.js:5]: Just a comment.

File: Header.js [hunk 6]
```diff
@@ -1,5 +1,10 @@
...
+      <h1>{ children }</h1>
+      <nav></nav>
+    </header>
+  );
```
Comment (stefantsonev2019) [inline Header.js:6]: Maybe we should stop using semicolons?

**Review by stefantsonev2019 — APPROVED** [review]

File: Root.js [hunk 7]
```diff
@@ -7,7 +7,7 @@ function App() {
     <div>
       <Header>Hello world</Header>
```
Comment (stefantsonev2019) [inline Root.js:8]: Update the copy please.

**Review by stefantsonev2019 — CHANGES_REQUESTED** [review]
Just one thing.

**Comment by krasimir** [comment]
Testing!!!!

**Comment by krasimir** [comment]
Testing 2

**Comment by krasimir** [comment]
Testing 3

**Comment by krasimir** [comment]
Here we go!

**Comment by krasimir** [comment]
Some markdown here:

```js
var test = 20;
```

**Comment by krasimir** [comment]
OK

**Comment by krasimir** [comment]
This time it works.

File: Content.js (hunk 5 above)
Comment (krasimir) [reply Content.js]: test

**Review by krasimir — COMMENTED** [review]
comment

File: Content.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+/* A
+comment here
+*/
+
+import React from 'react';
+
+export default function Content({ children }) {
+  return (
+    <section>
+      { children }
```
Comment (krasimir) [inline Content.js:10]: doze

File: Root.js (hunk 7 above)
Comment (krasimir) [reply Root.js:8]: Not sure.

File: LICENSE [hunk 9]
```diff
@@ -1,21 +0,0 @@
...
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
```
Comment (krasimir) [inline LICENSE:21]: Why's this deleted?

File: Content.js (hunk 5 above)
Comment (krasimir) [reply Content.js:5]: React is cool!

**Pull request merged**